db/migrate/20160829134746_remove_atomic_container_providers.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveAtomicContainerProviders < ActiveRecord::Migration[5.0]
+  include MigrationHelper
+
+  NAME_MAP = Hash[*%w(
+    ManageIQ::Providers::Atomic::ContainerManager                                 ManageIQ::Providers::Openshift::ContainerManager
```

Comment:
You can leave it...this is one of those cases where readability trumps rubocop's rules